src/main/java/bio/terra/cli/serialization/userfacing/resource/UFGcpDataprocCluster.java

```diff
@@ -0,0 +1,179 @@
+package bio.terra.cli.serialization.userfacing.resource;
+
+import static bio.terra.cli.businessobject.resource.GcpDataprocCluster.JUPYTER_LAB_COMPONENT_KEY;
+
+import bio.terra.axonserver.model.ClusterInstanceGroupConfig;
+import bio.terra.axonserver.model.ClusterLifecycleConfig;
+import bio.terra.axonserver.model.ClusterMetadata;
+import bio.terra.axonserver.model.ClusterStatus;
+import bio.terra.cli.businessobject.resource.GcpDataprocCluster;
+import bio.terra.cli.serialization.userfacing.UFResource;
+import bio.terra.cli.utils.UserIO;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.io.PrintStream;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * External representation of a workspace GCP Dataproc cluster resource for command input/output.
+ *
+ * <p>This is a POJO class intended for serialization. This JSON format is user-facing.
+ *
+ * <p>See the {@link GcpDataprocCluster} class for a Dataproc cluster's internal representation.
+ */
+@JsonDeserialize(builder = UFGcpDataprocCluster.Builder.class)
+public class UFGcpDataprocCluster extends UFResource {
+  public final String projectId;
+  public final String region;
```

Comment:
region is already part of parent class UFResource, please remove this from entire class